### DISCUSSION TITLE
No autoupdate?

### DISCUSSION THREAD

**Discussion by marissa999** [coder/code-server#2127] [Q&A] [answered]
I am pretty sure that code-server has an auto-update-feature? I could find it being mentioned in a few issues, but I was not able to find it in any guide (seems like there should be a popup in the bottom right?).

code-server --version
info  Using config file ~/.config/code-server/config.yaml
3.4.1 48f7c2724827e526eeaa6c2c151c520f48a61259

It seems like the current version is 3.5.0? What is the best way to update code-server? I run code-server with my own script and systemd-service file bc i need to source a few extra variables. Is that maybe related to it?

**Comment by marissa999** [comment]
https://github.com/cdr/code-server/issues/2131

**Comment by code-asher** [comment, marked as answer]
Ah yeah we stripped out the auto-updates in favor of using system package managers or npm/yarn or our install script.

The easiest way to update is to install it again the same way you did the first time. The extensions and settings and all that are in a separate directory so they'll be preserved. So for example if you used our install script you'd need to run that again.

I'd like to add update notices to code-server though that will tell you how to update. We could also add something to the FAQ.
